sources/packages/forms/src/form-definitions/institutionlocation.json

```diff
@@ -0,0 +1,1613 @@
+{
+    "_id": "612fa7e794d56e2366d54163",
```

Comment:
Please remove the id since it makes sense only in the environment where it was generated.